src/Components/GEOSldas_GridComp/GEOSlandpert_GridComp/GEOS_LandPertGridComp.F90

```diff
@@ -983,8 +983,9 @@ subroutine Initialize(gc, import, export, clock, rc)
        endif
 
        call ESMF_CFIOStrTemplate(rst_fname, trim(adjustl(rst_fname_tmp)),'GRADS', xid = trim(id_string), stat=status)
+       inquire(file=rst_fname, exist=f_exist)
```

Comment:
@weiyuan-jiang : One more thing... Should we restrict this "inquire()" to the Root node?  That is, replace this line with:
`if ( IAmRoot)  inquire(file=rst_fname, exist=f_exist)`
I suppose there's no need for all processors to send off their own inquiry to the file system.

Reply:
If we restrict `inquire()` to Root, then we'd also have initialize `f_exist = .true.` further up, I think.